### ISSUE TITLE
Adventure 001 Day 07 Retrospective

### ISSUE THREAD

**Issue by patmaddox** [RubySteps/21-day-challenge#400]
Day 07 is done! Please see #443 for the Day 08 retrospective

**Comment by amaliahern** [comment]
Today I learnt about shovel operator. I want to do more of basic exercises of ruby monk to improve my Ruby knowledge. I would like to try to fix my kata because it is red. :-(

**Comment by Dino-Herbert** [comment]
Learned that I can "puts" to a file in addition to the terminal screen....nice

**Comment by patmaddox** [comment]
I learned that I need to take at least one day off per week ;)

**Comment by kotp** [comment]
I learned today that I am grounded from the computer today, due to a birthday party.

**Comment by Dino-Herbert** [comment]
@patmaddox  burn out is real....let me know if I can help in some small way
@kotp lol...slowly step away from the keyboard and no one gets hurt...

**Comment by kotp** [comment]
@Dino-Herbert indeed... Fortunately, my PR is done for this 24 hour period, and I have time after midnight tomorrow to complete the task. :)  I like the [RSST](http://www.rubysteps.com/rst/) zone.

**Comment by tommycarney** [comment]
I learned that I can learn a ton by creating rails apps to test a simple idea. I learnt this from "Practicing Rails". 

I'm also learning a ton about creating amazing online courses from Pat Maddox. This is seriously one of the most engaging "course" I've done. Reminds me a lot of Tiny Habits by BJ Fogg

**Comment by zacscodingclub** [comment]
I learned quite a bit about working with git through the Terminal.  I had completed some tutorials in the past, but never quite understood it.  Not saying I have it completely figured out now, but while getting Day 6 and 7 updated, I had to clone a fork, add the files for day 6, push a commit, create a pull request, create and add files for day 7, push a commit (only to find out there was a conflict), then git pull to fetch and merge the commits, push that up, and create a pull request.
